.yarn/versions/de4a9a51.yml

```diff
@@ -0,0 +1,4 @@
+releases:
+  "@data-wrangling-components/core": patch
+  "@data-wrangling-components/react": major
```

Comment:
Let's set this to "minor" for now, since you are adding new verbs to the API but not disrupting the previous API (i.e., it is still backwards compatible)